tsi_app/m85/overlay_deferred.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_LOG_MODE_IMMEDIATE=n
+CONFIG_LOG_MODE_DEFERRED=y
+CONFIG_LOG_BUFFER_SIZE=1024
+CONFIG_LOG_PROCESS_THREAD=y
+CONFIG_LOG_PROCESS_THREAD_SLEEP_MS=100
+CONFIG_LOG_PROCESS_TRIGGER_THRESHOLD=2
+CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=4096
```

Comment:
For now this stack size and buffer size should be ok, but we may have to keep an eye for it as it may need increasing as we only have 2 MB SRAM.